skfda/exploratory/visualization/_magnitude_shape_plot.py

```diff
@@ -131,20 +134,24 @@ class MagnitudeShapePlot:
                         [-1. ],
                         [-1. ],
                         [-1. ]]]),
-                grid_points=(array([ 0.,  2.,  4.,  6.,  8., 10.]),),
+                grid_points=(array([  0.,   2.,   4.,   6.,   8.,  10.]),),
                 domain_range=((0.0, 10.0),),
-                ...),
+                dataset_name=None,
```

Comment:
Why did you add everything?

Reply:
because due to the changes it was now printing it, and if i didnt put it would be me an error as the expected was different to the output

Reply:
It also printed it before the changes. The ellipsis should match anything, though.

Reply:
ok, ellipsis is working mb